ttnn/cpp/pybind11/device.cpp

```diff
@@ -621,6 +622,11 @@ void device_module(py::module& m_device) {
         +------------------+----------------------------------+-----------------------+-------------+----------+
     )doc");
 
+    m_device.def(
+        "get_arch_name",
+        []() -> std::string { return tt::tt_metal::experimental::hal::get_arch_name(); },
```

Comment:
```suggestion
        &tt::tt_metal::experimental::hal::get_arch_name,
```

@yan-zaretskiy @dmakoviichuk-tt